src/components/Teams.js

```diff
@@ -0,0 +1,17 @@
+const Teams = (props) => {
+  /* eslint-disable */
+  const teams = props.teams;
+  return (
+    <div className="teams">
+      {teams.map((team) => (
+        <div key={team.id} className="team">
+          <p className="fullname">{team.full_name}</p>
+          <p className="city">{team.city}</p>
+          <p className="conference">{team.conference}</p>
+        </div>
+      ))}
+    </div>
```

Comment:
- [ ] Good job being able to dynamically display the details for a specific team. There is something very small remaining design style-wise. Kindly note that in the project requirements, the design has an alternating stle just like in the home page and the list has a "Zebra crossing" like effect on the design.
- [ ] Kindly also make the details scale the whole page so that the rest of the page has a pink theme and not a white background for areas where the teams are less and don't fit the whole page

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/203929585-1d3091dc-7a8e-44a0-a7b1-6137700805bd.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/203929658-626b774c-d6b2-4fcd-b8b8-8c102aa3911f.png)

Reply:
- [ ] Please address this as requested